packages/blocks/src/code-block/code-block.ts

```diff
@@ -169,6 +171,15 @@ export class CodeBlockComponent extends NonShadowLitElement {
   @state()
   private _wrap = false;
 
+  @state()
```

Comment:
I guess this is not a `@state` since it won't be reassigned or trigger reactive update?